gettsim/data/soli_st.yaml

```diff
@@ -1,43 +1,51 @@
-solifreigrenze:
+solisatz:
   name:
-    de: Soli-Freigrenze  (§ 3 (3) Nr. 2 SolzG)
+    de: Zuschlagsatz
+    en: Tax rate
+  description:
+    de: Höhe des Solidaritätszuschlags auf die Bemessungsgrundlage. § 4 SolZG in den Jahren 1991 und 1992. § 4 SolZG 1995 ab 1995.
     en:
+  values:
+    1991-01-01:
+      value: 0.0375
+      note: Artikel 1 G. v. 24.06.1991 BGBl. I S. 1318.
+    1995-01-01:
```

Comment:
wasn't it zero in the meantime?

Reply:
No.

If you refer to the 1993 and 1994 years: There was no Soli at all.

> Note there are two different Solis:
> 
>  1. [Solidaritätszuschlaggesetz (SolZG)](https://offenegesetze.de/veroeffentlichung/bgbl1/1991/38#page=14) in 1991 and 1992.
> 
>  2. [Solidaritätszuschlaggesetz 1995 (SolZG 1995)](https://offenegesetze.de/veroeffentlichung/bgbl1/1993/30#page=43) since 1995.
> 
> 
> There is no Soli in 1993 and 1994.
> 


0.0375 is the parameter for the SolZG. I just re-use this parameter `solisatz` for the SolZG 1995. From 1995 this parameter is 0.55. I could also make this a second parameter.

Reply:
I'm saying it would be cleaner to set it to zero until SolZG 1995 was introduced.

Reply:
I disagree. There is no value 0 for that parameter `solisatz` for these years because there is no Soli law defined for that year. A Soli should not be calculated for 1993 and 1994, or 1990 if there is no law saying so. And this is handled in the python source code.